driver-async/src/main/com/mongodb/async/client/WriteOperationThenCursorReadOperation.java

```diff
@@ -22,24 +22,23 @@
 import com.mongodb.binding.AsyncWriteBinding;
 import com.mongodb.operation.AsyncReadOperation;
 import com.mongodb.operation.AsyncWriteOperation;
-import com.mongodb.operation.FindOperation;
 
-class AggregateToCollectionThenFindOperation<T> implements AsyncReadOperation<AsyncBatchCursor<T>> {
+class WriteOperationThenCursorReadOperation<T> implements AsyncReadOperation<AsyncBatchCursor<T>> {
```

Comment:
Generalized this since we no longer have the concrete operation classes to base it on.

Reply:
Ack